Parser/StarmonMiniParse.m

```diff
@@ -373,6 +386,7 @@
             data.(var).applied_offset = applied_offset;
             
         case 'Depth(m)'
+        case 'Depth0x28m0x29'
```

Comment:
Hmm, yes, matlab is not like C, where the the case drops through without a break